docs/UserGuide.md

```diff
@@ -156,14 +159,15 @@ This practice helps prevent data loss and ensures a proper shutdown of the appli
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </div>
 
+> [Back to Table of Contents](#table-of-contents)
```

Comment:
can make table of contents hyperlink formatted like others pls

Reply:
Changed all to blue